apps/financial-aid/web-veita/src/routes/Application/application.tsx

```diff
@@ -287,7 +287,7 @@ const ApplicationProfile = () => {
     )
   }
   if (loading) {
-    return <LoadingDots />
+    return <ApplicationSkeleton />
```

Comment:
Why do we sometimes use a ternarny for the skeleton load and sometimes return it?